api/src/main/java/marquez/graphql/MarquezGraphqlServletBuilder.java

```diff
@@ -0,0 +1,226 @@
+package marquez.graphql;
+
+import static graphql.schema.idl.TypeRuntimeWiring.newTypeWiring;
+
+import com.codahale.metrics.MetricRegistry;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+import com.smoketurner.dropwizard.graphql.CachingPreparsedDocumentProvider;
+import com.smoketurner.dropwizard.graphql.GraphQLFactory;
+import graphql.execution.preparsed.PreparsedDocumentProvider;
+import graphql.kickstart.execution.GraphQLQueryInvoker;
+import graphql.kickstart.servlet.GraphQLHttpServlet;
+import graphql.schema.Coercing;
+import graphql.schema.CoercingParseLiteralException;
+import graphql.schema.CoercingParseValueException;
+import graphql.schema.CoercingSerializeException;
+import graphql.schema.GraphQLScalarType;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.idl.RuntimeWiring;
+import graphql.schema.idl.SchemaGenerator;
+import graphql.schema.idl.SchemaParser;
+import graphql.schema.idl.TypeDefinitionRegistry;
+import java.net.URL;
+import java.time.ZonedDateTime;
+import java.util.Map;
+import java.util.UUID;
+import lombok.SneakyThrows;
+import org.jdbi.v3.core.Jdbi;
+
+public class MarquezGraphqlServletBuilder {
```

Comment:
_Minor_: We'll want to be consistent with lowercase vs uppercase in graphql class names. I'd favor all lowercase :)